### ISSUE TITLE
Global notifications

### ISSUE THREAD

**Issue by taschmidt** [PeteGoo/tcSlackBuildNotifier#13]
Can we get a way to set up notifications globally?  If I go to the Slack tab on the `<Root Project>`, it's blank.

**Comment by sergiivolchkov** [comment]
+1, would be very useful.

**Comment by Youkai-no-nii-chan** [comment]
+1. We have over 20 projects so it rather tiresome to set notifications for all of them.

**Comment by maartenc** [comment]
+1

**Comment by gluckett** [comment]
+1

**Comment by mr-fixit** [comment]
Is this request not satisfied by the page at `http://teamcity.YOURTEAM.link/admin/admin.html?item=slackNotifications` ?

**Comment by sergiivolchkov** [comment]
@mr-fixit no, it's not. The admin page has Slack connection settings and notification features.

Project-specific config pages like `http://teamcity.YOURTEAM.link/slacknotifications/index.html?projectId=YOURPROJECT` is where build events triggering notifications are configured. Those settings can be redefined for individual builds or subprojects, but it is not possible to configure them for Root project to be used as a template for all projects in TC instance.

**Comment by mr-fixit** [comment]
Thanks, @sergiivolchkov. I'd forgotten trudging around the UI, creating SlackNotifications for every project.  Eventually I created a script to ask TeamCity-API for every active project, then generated the URLs for the right page.  Still painful, but less.

**Comment by whereisaaron** [comment]
In a per-user configuration you can watch the root project and thus get global notifications for a user or group of users. However you can't specify a channel as a user name.

If it were possible for users to specify a channel as their user name, this might be another way to achieve global notifications to a channel?

**Comment by aaron-0** [comment]
I'd love to see global notifications as well. Additionally it would be good if I could set a filter on name. For instance I want to notify a specific channel whenever one of my deployment builds fails.  They are all consistently named so easy to filter.
